docs/dev/news/5882.fix

```diff
@@ -0,0 +1 @@
+Enable using reveal statement expression inside witness expressions
```

Comment:
```suggestion
Enable using reveal statement expression inside witness expressions when the new resolver is activated with `--type-system-refresh`
```

Reply:
I removed this limitation in a previous PR